Bond/Extensions/iOS/UITableView+Bond.swift

```diff
@@ -106,10 +113,13 @@ private class BNDTableViewDataSource<T>: NSObject, UITableViewDataSource {
   private class func applySectionUnitChangeSet(changeSet: ObservableArrayEventChangeSet, tableView: UITableView, dataSource: BNDTableViewProxyDataSource?) {
```

Comment:
Hmm, we should not do reload in this method (`applySectionUnitChangeSet`) nor in the method `applyRowUnitChangeSet` because in case of batch updates reloadData could be called multiple times unnecessarily.